src/components/reviewMatches/Dialog.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Dialog as MuiDialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Typography
+} from '@mui/material'
+
+interface DialogProps {
+  buttons: JSX.Element[]
+  title: string
+  subTitle?: string
+  content: JSX.Element | string
+  onOpen: boolean
+  onClose: () => void
+  fullWidth?: boolean
+  maxWidth?: 'sm' | 'md' | 'lg'
+}
+
+const Dialog: React.FC<DialogProps> = ({
+  buttons,
+  title,
+  subTitle,
+  content,
+  onOpen,
+  onClose,
+  fullWidth = false,
+  maxWidth = 'sm'
+}) => {
+  return (
+    <MuiDialog
+      open={onOpen}
+      onClose={onClose}
+      fullWidth={fullWidth}
+      maxWidth={maxWidth}
+    >
```

Comment:
@IlyesBenAmara @Lumiere-Lodi I think there's two components we are using Modal and MuiDialog. What's the difference between the two ?

Reply:
Not much of different. I look at them as Modal is a blank white paper that you can customize however you want and Dialog is a bit high level with some customization done for you (Dialog is also using Modal in the background). 

You can also see the difference [here](https://mui.com/material-ui/react-modal/#:~:text=If%20you%20are%20creating%20a%20modal%20dialog%2C%20you%20probably%20want%20to%20use%20the%20Dialog%20component%20rather%20than%20directly%20using%20Modal.%20Modal%20is%20a%20lower%2Dlevel%20construct%20that%20is%20leveraged%20by%20the%20following%20components%3A) in their site

For something that requires a title, content, and action buttons. A modal makes sense because it has the skeleton done for you already. If it is more complicated than that then we can use Modal. 

@IlyesBenAmara What do you think?

Reply:
I used the modal component for the `edit golden record` feature and for the `refine search` search.

Reply:
@IlyesBenAmara @Lumiere-Lodi , first PR gets merged will rule over which one to use 🏎️  😄